package.json

```diff
@@ -134,7 +134,7 @@
       "prefer-const": 1,
       "prefer-spread": 2,
       "strict": [2, "global"],
-      "indent": [1, 2]
+      "indent": [1, 2, {"SwitchCase": 1}]
```

Comment:
`SwitchCase` means that we do:
```
switch () {
  case 'foo':
    return x;
}
```

instead of:
```
switch () {
  case 'foo':
  return x;
}
```